test_schemas/en/test_generate_date_range.json

```diff
@@ -0,0 +1,223 @@
+{
```

Comment:
We group schemas with a prefix relevant to the concept tested in the schema, so for this it would be like `test_placeholder_date_range` etc.